src/model/src/o3totcol.f

```diff
@@ -115,10 +114,10 @@ subroutine o3totcol ( latitude, longitude, jdate, jtime, ozone )
           call m3exit ( pname, jdate, 0, xmsg, xstat1 )
         end if
 
-        ! read nlat, nlon (Wei Li)
+        ! read nlat, nlon
         rewind( tmunit )
-        read( tmunit, *) label, nlat
-        read( tmunit, *) label, nlon
+        read( tmunit, '(5x,i7)') label,nlat
+        read( tmunit, '(5x,i7)') label,nlon
```

Comment:
This change will cause run-time error, as variable "label" is defined as "character( 8 ) :: label" and '(5x,i7)' only supply one variable's input format instead of two. It's better to reverse back

Reply:
thanks! reversed